app/assets/javascripts/table_scroll.js

```diff
@@ -0,0 +1,17 @@
+var scrolling = false;
+jQuery(document).on('ready page:load',function() {
+    $("#course_table").bind('mousewheel DOMMouseScroll', function(event) {
+        event.preventDefault();
+        if (scrolling)
+        {
+            return;
+        }
+        scrolling = true;
+        var delta = (event.originalEvent.wheelDelta
+            ? event.originalEvent.wheelDelta / 120
+            : event.originalEvent.detail / -3);
+        $('#course_table').animate({
```

Comment:
Try this animation. It works more smoothly, than in first commit, but it can ignore some wheel events and it may seen that scrolling is lagging.